scenarios/README.md

```diff
@@ -401,6 +401,41 @@ The scenario named `custom` can be used to pass any custom filter variable like
 crank --config https://github.com/aspnet/benchmarks/blob/main/scenarios/dotnet.benchmarks.yml?raw=true --scenario custom --profile aspnet-perf-win --variable filter=*LinqBenchmarks*
 ```
 
+## HTTP client benchmarks
+
+These scenarios are running various HTTP client benchmarks.
+
+### Sample
+
+```
+crank --config https://raw.githubusercontent.com/CarnaViire/Benchmarks/http3-benchmarks/scenarios/httpclient.benchmarks.yml --scenario httpclient-kestrel-get --profile aspnet-perf-lin
+```
+*TODO: update to `https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/httpclient.benchmarks.yml`*
+
+### Available scenarios
+
+- `httpclient-kestrel-get`: Execute GET requests to the server. Client is HttpClient, server is Kestrel (minimal API).
+
+#### Arguments
```

Comment:
How do I control TLS vs non-TLS?

Reply:
Since the idea of the benchmark originated from HTTP/3 which is always TLS, I didn't add the parameter here. But it makes sense, so I should add it, now or in the future. Do you think it's better to add right away in this PR?

Reply:
I think non-TLS scenarios are so basic and relevant that they should be added in this PR.